app/assets/javascripts/components/emails/draft_content.es6.jsx

```diff
@@ -0,0 +1,81 @@
+class DraftContent extends Component {
+
+  // --------------------------------------------------
+  // Props
+  // --------------------------------------------------
+  static get propTypes() {
+    return {
+      id: React.PropTypes.number.isRequired,
+      errors: React.PropTypes.array,
+      margin: React.PropTypes.bool,
+      value: React.PropTypes.string,
+    };
+  }
+
+  static get defaultProps() {
+    return {
+      errors: [],
+      focus: false,
+      margin: false,
+      value: '',
+    };
+  }
+
+  // --------------------------------------------------
+  // Styles
+  // --------------------------------------------------
+  get styles() {
+    return {
+      container: Object.assign(
+        {},
+        {
+          display: 'flex',
+          flexFlow: 'column',
+          alignSelf: 'stretch',
+        },
+        this.props.margin && { marginTop: '12px' }
+      ),
+      error: {
+        color: StyleConstants.colors.red,
+      },
+    };
+  }
+
+  // --------------------------------------------------
+  // Lifecycle
+  // --------------------------------------------------
+  componentDidMount() {
+    var node = ReactDOM.findDOMNode(this.refs.input);
+    node.oninput = (event) =>
+    DraftActions.storeAttribute('content', event.target.value,
+                               this.props.id);
+  }
+
+  // --------------------------------------------------
+  // Render
+  // --------------------------------------------------
+  renderError() {
```

Comment:
there's no server validation but isn't this already set up to work correctly if there was?